Makefile

```diff
@@ -325,7 +379,9 @@ clean:
 	cd Firefox && $(MAKE) clean
 	cd Webmin && $(MAKE) clean
 	cd Chromium && $(MAKE) clean
-	rm -f scap-security-guide.spec
+	rm -f $(RPM_SPEC)
```

Comment:
Hi @konstruktoid, Why not using dh_make to support cleaner debian source package ?
This can be donne using : 

dh_make -c custom --copyrightfile ../LICENSE -e __MAINTAINER_MAIL__ -n -p scap-security-guide___VERSION__ -y -C i

This will allow the generation of the debian/ subdir. control and changelog files are templated and will need to be updated for their message content and target DIST release only, allowing the usage of tools such as gbp or dpkg-buildpackage directly, generating proper source+bin package.

Only the debian/scap-security-guide.install file has to be generated, containing which files need to be added to the bin package depending on the target distribution, as you has started in your script.